packages/test/test-end-to-end-tests/src/test/detachedContainerTests.spec.ts

```diff
@@ -817,7 +817,11 @@ describeFullCompat("Detached Container", (getTestObjectProvider) => {
 				if (runtimeMessage === false) {
 					return;
 				}
-				const envelope = message.contents.contents.content;
+				const envelope = (
+					(message.contents as { contents: unknown }).contents as {
```

Comment:
Looks like casting overkill. Wouldn't `= (message.contents as any).contents.content;` do the trick?

Reply:
Good catch; the simpler form worked.